lib/test.gi

```diff
@@ -108,11 +108,15 @@ InstallGlobalFunction(RunTests, function(arg)
   cmp := [];
   times := [];
   ttime := Runtime();
+  nrlines := pos[Length(pos) - 1];
```

Comment:
This fails if `pos` is empty or contains only one entrye, e.g. if the test file contains no actual tests -- which is the case for `tst/teststandard/bugfix.tst`.  In that case, `pos = [ [ 1, 2, 3, 4, 5, 6, 7, 8 ] ]`.